content/blog/20240430_fluentd-v1.17.0-has-been-released.md

```diff
@@ -35,14 +35,36 @@ Please see [the document](https://docs.fluentd.org/input/tail#glob_policy) and [
 
 ### `out_http`: Support AWS Signature Version 4 authentication
 
+In this release, we added a new option `aws_sigv4` for the [method](https://docs.fluentd.org/output/http#method) setting of [out_http](https://docs.fluentd.org/output/http) plugin.
+
+By using this option, `out_http` can use [AWS Signature Version 4](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
+
+For example, this allows `out_http` to write to [Amazon OpenSearch Ingestion](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ingestion.html).
+
+Please see [the document](https://docs.fluentd.org/output/http#method) and [#4459](https://github.com/fluent/fluentd/pull/4459) for more information.
+
 ### `out_http`: Add option to reuse connections
 
+In this release, we add a new option [reuse_connections](https://docs.fluentd.org/output/http#reuse_connections) for [out_http](https://docs.fluentd.org/output/http) plugin.
+
+This option will improve performance of `out_http`.
```

Comment:
it's a same meaning but we'd like to say:

```suggestion
This option will improve throughput of `out_http`.
```
but the original version of the sentence is also reasonable to use in the documentation.

Reply:
Thanks!